src/PerformanceTests/readme.md

```diff
@@ -22,10 +22,18 @@ Table of contents
 
 
 
-# Prerequisites for running tests
+# Prerequisites
 
-- Making sure that connection string to MS SQL DB is correct
-- Creating empty database manually before first run
+Tools for development and testing:
+
+- Visual Studio 2017 (tested with 15.3.5)
+- [Slow Cheetah for vs2017](https://marketplace.visualstudio.com/items?itemName=VisualStudioProductTeam.SlowCheetah-XMLTransforms)
+
+
+Configuration and environment:
+
+- Make sure that MSSQL connection strings are correct
+- A (empty) MSSQL database, this needs to be created manually
```

Comment:
name of the database?

Reply:
That depends on your personal connection string.